### PR TITLE
fix: replace INFO with IGNORE in zap rules

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

replace INFO with IGNORE for the ZAP scanning rules. 

Looks like the INFO setting does not prevent the alert from being thrown. Replace with IGNORE.

## Testing

test locally using Zap Docker run on cloud dev. 

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [ ] docs

OR

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: .zap/rules.conf [hunk 1]
```diff
@@ -1,9 +1,14 @@
-# Mark the following rules as INFO
+# Mark the following rules as IGNORE
```
Comment (gao-sun) [inline .zap/rules.conf:1]: the PR title and description are ambiguous - should specify it's updating zap rules

File: .zap/rules.conf [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
+90034	IGNORE	(Cloud Metadata Potentially Exposed - Active/release)
 
-10096	INFO	(Timestamp Disclosure - Passive/release)
-10063-1	INFO	(Permissions Policy Header Not Set - Passive/beta)
-10055-4	INFO	(CSP - Wildcard Directive)
-40039	INFO	(Web Cache Deception)
+# Not applicable to the cloud appliaction
+10096	IGNORE	(Timestamp Disclosure - Passive/release)
+
+# TODO
```
Comment (gao-sun) [inline .zap/rules.conf:9]: What does the TODO mean?

File: .zap/rules.conf [hunk 3]
```diff
@@ -1,9 +1,14 @@
...
+10063	IGNORE	(Permissions Policy Header Not Set - Passive/beta)
+
+# The applicationInsights endpoint will be removed
+10055	IGNORE	(CSP - Wildcard Directive)
+40039	IGNORE	(Web Cache Deception)
```
Comment (gao-sun) [inline .zap/rules.conf:14]: Is this also fore appinsights?

File: .zap/rules.conf (hunk 2 above)
Comment (simeng-li) [reply .zap/rules.conf:9]: need to verify the alert

File: .zap/rules.conf (hunk 1 above)
Comment (simeng-li) [reply .zap/rules.conf:1]: updated

File: .zap/rules.conf (hunk 3 above)
Comment (simeng-li) [reply .zap/rules.conf:14]: o, put it under the wrong place/comment. Tested not an issue in our cloud environment.  Updated,

**Review by gao-sun — APPROVED** [review]

**Pull request merged**